snippets/head-config.liquid

```diff
@@ -138,6 +138,10 @@
   src='https://unpkg.com/ionicons@latest/dist/ionicons/ionicons.esm.js'
 ></script>
 
+{% javascript %}
+  const ratings = "{{ 'reviews.ratings' | t }}";
```

Comment:
we still have a bug related to the reviews try to add a product column and product reviews and single product section all of them with different product you will notice that all of them have the same number of rating because all of them have just took the reviews of the first product you added

Reply:
we already introduce this issue since we don't have reviews provider for each product
ticket link: `https://youcanshop.atlassian.net/browse/YSHOP2-175`